components/admin_console/license_settings/license_settings.test.tsx

```diff
@@ -112,24 +113,26 @@ describe('components/admin_console/license_settings/LicenseSettings', () => {
             upgradeToE0Status: jest.fn().mockImplementation(() => Promise.resolve({percentage: 0, error: null})),
         };
         const props = {...defaultProps, enterpriseReady: false, actions};
-        const wrapper = shallow(<LicenseSettings {...props}/>);
+        const wrapper = shallow<LicenseSettings>(<LicenseSettings {...props}/>);
 
         expect(actions.getLicenseConfig).toBeCalledTimes(1);
         expect(actions.upgradeToE0Status).toBeCalledTimes(1);
         actions.upgradeToE0Status = jest.fn().mockImplementation(() => Promise.resolve({percentage: 1, error: null}));
 
         const instance = wrapper.instance();
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
`interval` is private and TS is complaining about it. 
I also didn't want to change the component so I put comment to ignore typescript, please let me know if there is any other way to handle this.

Reply:
nit: brackets notation `instance['interval']` or reflect api  (`Reflect.set(instance, 'interval', ...);`) or just by disabling type checks (`(instance as any).interval`) should do the trick. However, I am ok with this approach too ;)

Reply:
Thanks for the suggestion, I will keep in mind in future PRs